tests/test_hear_keyboard_interrupt.py

```diff
@@ -74,13 +74,20 @@ def test_hear_keyboard_interrupt(self):
 		try:
 			self.assertIsNotNone(_fixture_port_num)
 			self.assertEqual(type(_fixture_port_num), type(int(0)))
-			_fixture_cmd = str("{} -m coverage run -p --context=Integration").format(sys.executable)
+			_fixture_cmd = str("{} -m coverage run -p --context=Integration").format(
+				sys.executable
+			)
```

Comment:
This could also be improved as per #134 with F-strings.

```suggestion
			_fixture_cmd = f"{str(sys.executable)} -m coverage run -p --context=Integration"
```